app/src/pages/inside/dashboardItemPage/widgetsGrid/widgetsGrid.jsx

```diff
@@ -211,6 +211,7 @@ export class WidgetsGrid extends Component {
               isDraggable={this.props.isModifiable}
               isResizable={this.props.isModifiable}
               draggableHandle=".draggable-field"
+              useCSSTransforms={false}
```

Comment:
I read [here](https://github.com/STRML/react-grid-layout) that this approach can decrease our performance in other browsers (see the screenshot below).
![image](https://user-images.githubusercontent.com/31861805/58198398-d0d55c00-7cd6-11e9-8252-3102788ccdb0.png)
@DiscoElevator what you do think? Can we adjust this fix only for FireFox?

Reply:
Fixed with adding the ability to disable `useCSSTransforms` feature in Firefox browser only.